itests/org.openhab.core.binding.xml.tests/src/main/resources/test-bundle-pool/BundleInfoTest.bundle/ESH-INF/thing/thing-types.xml

```diff
@@ -46,7 +46,8 @@
 		<item-type>ColorItem</item-type>
 		<label>HUE Lamp Color</label>
 		<description>The color channel allows to control the color of the hue
```

Comment:
This part looks weired.

Reply:
I agree. If you try to reformat it inside Eclipse you get the same result. It is probably related to previous manual formatting - if you remove the previous line breaks and then reformat the output is as it is supposed to be.

Reply:
I changed it by hand.